docs/transit_database/transitdatabase.md

```diff
@@ -0,0 +1,80 @@
+
+* [Airtable Data Documentation Google Doc](https://docs.google.com/document/d/1KvlYRYB8cnyTOkT1Q0BbBmdQNguK_AMzhSV5ELXiZR4/edit#heading=h.u7y2eosf0i1d) - documentation of specific fields in Airtable
+* [California Transit Data - Operating Procedures Google Doc](https://docs.google.com/document/d/1IO8x9-31LjwmlBDH0Jri-uWI7Zygi_IPc9nqd7FPEQM/edit#) -  outlines the processes by which Airtable data is maintained
+
+In addition, some documentation is available automatically within Airtable (these require Airtable authentication to access):
+* Airtable creates an API documentation page for each base (for example, [here is the page for California Transit](https://airtable.com/appPnJWrQ7ui4UmIl/api/docs)). This page provides technical information about field types and relationships. Airtable does not currently have an effective mechanism to programmatically download your data schema (they have paused issuing keys to their metadata API).
+* When looking at a base, there is an `Extensions` tab at the far upper right corner (below the share, notifications, and user icons). If you click that, an extensions sidebar will open. In that sidebar, there is an extension called `Base schema` (you may have to open it fullscreen to actually see it.) This extension will let you see an auto-generated visualization of the technical relationships among fields in the base.
+
+Cal-ITP uses two main Airtable bases:
+
+| **Base** | **Description** |
+| :------------ | :-------------- |
+| [**California Transit**](#california-transit) | Defines key organizational relationships and properties. Organizations, geography, funding programs,  transit services, service characteristics, transit datasets such as GTFS, and the intersection between transit datasets and services.
+| [**Transit Technology Stacks**](#transit-technology-stacks) | Defines operational setups at transit provider organizations. Defines relationships between vendor organizations, transit provider and operator organizations, products, contracts to provide products, transit stack components, and how they relate to one-another.
+
+The rest of this page outlines stray technical considerations associated with Airtable and its ingestion into the data warehouse.
+
+## Primary Keys
+
+Airtable forces the use of the left-most field as the primary key of the database: the field that must be referenced in other tables, similar to a VLOOKUP in a spreadsheet. Unlike many databases, Airtable doesn't enforce uniqueness in the values of the primary key field.  Instead, it assigns it an underlying and mostly hidden unique [`RECORD ID`](https://support.airtable.com/hc/en-us/articles/360051564873-Record-ID), which can be exposed by creating a formula field to reference it.
+
+## Importing Airtable data into the Cal-ITP data warehouse
+
+We ingest data from Airtable into the Cal-ITP data warehouse. To see general information about the existing ingest processes, see [the pipeline architecture documentation](architecture-overview).
```

Comment:
I would turn...

> To see general information about the existing ingest processes

...into...

> To learn about about how data flows through the existing data ingest processes

...and perhaps link directly to the #data-flow heading in the architecture overview, rather than the top-level doc.

Reply:
Thanks, this will need to change based on some changes in #2847 that I think I'll need to merge first (data will have a dedicated page). Won't merge this PR until @evansiroky has reviewed.